pkg/ephemeral-disk-utils/utils.go

```diff
@@ -94,6 +94,7 @@ func FileExists(path string) (bool, error) {
 }
 func Md5CheckSum(filePath string) ([]byte, error) {
 
+	// #nosec no risk for path injection: only used to calculate MD5 of files
```

Comment:
I wasn't able to find a issue here. Do you think we can improve the Gosec rule?

Reply:
The Open also need to be cleaned